docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,119 @@
+
+We have this policy to allow a reporter to request the *delisting* of forks that are old and outdated.
+
+
+High level overview
+~~~~~~~~~~~~~~~~~~~
+
+The process at a high level looks like:
+
+* A reporter contacts us about a project they think is outdated and unofficial
+* A Read the Docs team member evaluates it to make sure it's outdated and unofficial, according to this policy
+* We delist this project from search results and send an email to owners of the Read the Docs project
+* If a project owner objects, we evaluate their evidence and make a final decision
+
+
+Definitions
+-----------
+
+
+Unofficial projects
+~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unofficial* when it is not linked to or mentioned in ANY of these places:
+
+* Websites and domains associated with the project
+* The project's primary repository -- README files, repository description, or source code
+
+
+Unmaintained projects
+~~~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unmaintained* when ANY of the following are met:
+
+* The configured version control repository is unreadable. This can happen if the repository is deleted, credentials are broken or the Git host is permanently unresponsive.
+* The project is only serving content from releases and commits 6 months or older.
+* All builds have failed for more than 6 months.
+
+
+Implementation
+--------------
+
+
+Requesting a project be delisted
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can request that we delist an outdated, unmaintained documentation by contacting our :doc:`/support`.
+
+Please include the following information:
+
+.. code-block:: text
+
+  URL of unofficial and unmaintained documentation project: ...
+  URL of official documentation (if any): ...
+  URL of official project website (if any): ...
+  URL of official project repository (if any): ...
+
+  Describe attempts of reaching the owner(s) of the documentation project:
+  ...
+
+
+Delisting
+~~~~~~~~~
+
+Projects that are determined to be unmaintained_ and unofficial_ will have a ``robots.txt`` file added that removes them from all search results:
+
+.. code-block:: text
+
+  # robots.txt
+  User-agent: *
+  # This project is delisted according to the Policy for Unofficial and Unmaintanied Projects
+  # https://docs.readthedocs.io/en/stable/unofficial-projects.html
+  Disallow: /
+
+
+Projects will be delisted if they meet ALL of the following criteria:
+
+* The person who submits the report of the unmaintained_ and unofficial_ project also demonstrates failed attempts to contact the existing owners.
+* The project has been determined unmaintained_ and unofficial_ by the rules described above.
+* The project owners are notified by the Read the Docs team.
+* The core team does not have any additional reservations.
+* If any of the project owners respond, their response will be taken into account by the core team and it may be listed again if the criteria are met.
```

Comment:
The first and last item here are not criteria, they are process we are following. I think it's much clearer to break them out:

```suggestion
* The core team does not have any additional reservations.

The Read the Docs team will do the following actions when a project is delisted:

* Notify the Read the Docs project owners via email about the delisting.
* Add the ``robots.txt`` file to be served on the project domain.

If any of the project owners respond, their response will be taken into account, and the delisting might be reserved.
```